### ISSUE TITLE
In the model tree I can not use Select All option if the elements are hidden

### ISSUE THREAD

**Issue by BDavydov** [3drepo/3drepo.io#2078]
### Description
Select all function in the model tree does not work when elements are hidden. 

### Steps to replicate
- Log in to 3drepo.io
- Open any model
- Open the tree card
- Specify any search criteria to list multiple model elements
- In the options menu press Select All
- Hide selected model elements
- Clear the search filter
- Search for the same elements again
- In the options menu press Select All
- Objects are not selected and the list jumped to the bottom

### Pictures/ Gifs of the error
![Select All Hiden](https://user-images.githubusercontent.com/19904513/84400068-7e285400-abf9-11ea-8b0c-7916cab517c3.gif)

**Comment by m1chalkubiak** [comment]
Hi @BDavydov @carmenfan 
I tried to analyze what this issue looks like from the technical side and it seems to me that this work correctly. I already explain why.
What is visible in the panel is based on the data we receive from the Viewer, the items that are marked in the panel are in fact elements in the model that are highlighted, I did not go into it that deeply but I think that Carmen will be in this matter able to answer. Model elements that are not visible cannot be highlighted and thus cannot be displayed as selected in the tree panel. Unless it is a bug that has wrongly assumed the way this panel works. It seems to me to work fine.

In the code, it also seems that there are many places confirming this assumption, for example:
```js
	private handleSelectAllNodes = () => {
		this.props.selectNodes(this.props.visibleNodesIds);
	}
```
